testsuite/features/secondary/sle_minion_bootstrap_ssh_key.feature

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2016-2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Be able to bootstrap a Salt minion via the GUI using SSH key
+
+  Scenario: Delete SLES minion system profile before bootstrap test
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    And I cleanup minion "sle_minion"
+    Then "sle_minion" should not be registered
+
+  Scenario: Prepare the minion for SSH key authentication
+    When I backup the SSH authorized_keys file of host "sle_minion"
+    And I add pre-generated SSH public key to authorized_keys of host "sle_minion"
+  
+  Scenario: Bootstrap a SLES minion using SSH Key with wrong passphrase
+    Given I am authorized
+    When I go to the bootstrapping page
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "sle_minion" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I check radio button "SSH Private Key"
+    And I attach the file "ssh_keypair/id_rsa_bootstrap-passphrase_linux" to "privKeyFile"
+    And I enter "you-shall-not-pass" as "privKeyPwd"
```

Comment:
Do we have balrogs for our tests? ;-)

Reply:
Yes, and we need to make sure they don't pass ;)